### PR TITLE
[HAC-167] Embed email validation on landingpage

### PR DESCRIPTION
**Pull request by kkkowalski**
- Added `formsy-react` and `formsy-material-ui` packages
- Changed eslint rule: `jsx-boolean-value` to 0
- Enabled validation on field onBlur and after button press

### CODE REVIEW

File: src/apps/LandingRegister/LandingRegister.js [hunk 1]
```diff
@@ -58,11 +57,26 @@ export default Radium(React.createClass({
...
   },
 
+  disableSubmit() {
+    this.setState({
+      canSubmit: false
+    });
+  },
+
+  submitForm(model) {
+    console.log('Model: ', model);
```
Comment (patrykkopycinski) [inline src/apps/LandingRegister/LandingRegister.js]: do you need it?

Comment (kkkowalski) [inline src/apps/LandingRegister/LandingRegister.js]: console.log is temporary to show data access, will be replaced with calling Syncano webhook when all subpage styles are complete

Comment (patrykkopycinski) [inline src/apps/LandingRegister/LandingRegister.js]: :+1:

**Comment by kkotelczuk** [comment]
LGTM

File: src/apps/LandingRegister/LandingRegister.js [hunk 2]
```diff
@@ -57,10 +56,9 @@ export default Radium(React.createClass({
     };
   },
 
-  handleButtonPress() {
-    this.setState({
-      registerState: true
-    });
+  submitForm(model) {
+    console.log('Model: ', model);
```
Comment (mkucharz) [inline src/apps/LandingRegister/LandingRegister.js]: As far as I remember we are using in Syncano console.debug, which can be easily cleaned during production build.

Comment (patrykkopycinski) [inline src/apps/LandingRegister/LandingRegister.js]: yes, but there is a plan to use https://www.npmjs.com/package/debug instead

Comment (kkkowalski) [inline src/apps/LandingRegister/LandingRegister.js]: I assume that we can start using a console.debug, to keep one code convention.

**Comment by StudentDawid** [comment]
LGTM

**Pull request merged**